src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -73,8 +73,10 @@ export class DynamicFacetManager extends Component {
     })
   };
 
-  // Children DynamicFacet components of the DynamicFacetManager
-  private dynamicFacets: DynamicFacet[];
+  // Children DynamicFacet components of the DynamicFacetManager (at initialization)
+  private childrenDynamicFacets: DynamicFacet[];
```

Comment:
I think the names of these two properties could be improved so that the comments are not needed. Considering we are in the DynamicFacetManager file, I do not think `dynamic` is needed in the name. I would suggest `initialFacets` and either `responseFacets` or `recommendedFacets` or `suggestedFacets` or `receivedFacets`